lesson-4/README.md

```diff
@@ -0,0 +1,46 @@
+# Advanced React - Async
+
+## Goal - understand (unidirectional) data flow
+
+## TODO
+
+* [recap] Component's life cycle hooks recap
+* [recap] State & Props
+* [recap] Children + key (re-render)
+* [recap] Promises + Async
```

Comment:
async/await will be a new concept I think...

Reply:
I'll give it a try or will use promises ;)